index.html

```diff
@@ -168,5 +168,15 @@ <h2>Skills</h2>
                 <p class="arrow-right">></p>
             </div>
         </main>
+        <section id="contact">
+            <h1>Contact Me</h1>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <form action="https://formspree.io/f/xqkoqlbk" method="post">
+                <input id="text" type="text" placeholder="Enter your name" maxlength="30" required>
+                <input id="email" type="email" placeholder="Enter your email" required>
```

Comment:
- [ ] Great job integrating the form with Formspree service. :+1: Currently, only content of the textarea is getting collected by the service because it has a `name` attribute on it. Kindly add a `name` attribute on both inputs so their content can get collected by the service. ✍️